Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM runatlantis/atlantis:latest
+
+ENV TERRAGRUNT_VERSION=v0.21.7
+
+RUN curl -s -Lo terragrunt https://github.com/gruntwork-io/terragrunt/releases/download/${TERRAGRUNT_VERSION}/terragrunt_linux_amd64 && \
+    chmod +x terragrunt && \
+    mv terragrunt /usr/local/bin
+RUN apk add unzip make curl
```

Comment:
Just a small detail but we could move this up since `curl` is needed in the previous line